kerl

```diff
@@ -733,6 +743,63 @@ do_deploy()
     echo "kerl_deactivate"
 }
 
+is_valid_install_path()
+{
+    # don't allow installs into home directory
+    if [ "$1" = "$HOME" ]; then
+        echo "ERROR: You cannot install a build into $HOME. It's a really bad idea."
+        return 1
+    fi
+
+    # don't allow installs into .erlang because
+    # it's a special configuration file location
+    # for OTP
+    if [ "$1" = "$HOME/.erlang" ]; then
```

Comment:
```
$ cd
$ kerl install 18.3 .erlang
Installing Erlang/OTP 18.3 (18.3) in /home/vagrant/.erlang...
```

:-(

Maybe we should pre-process `$1` before calling this function, to make it an absolute path?